server/db/models/index.js

```diff
@@ -1,4 +1,7 @@
 const User = require('./user')
+const Product = require('./product')
+
+Product.belongsToMany(User, {through: 'ProductCategories'})
```

Comment:
Did you mean to create a relationship between Product + User? Or to Categories?

```suggestion
const Categories = require('./categories')
Product.belongsToMany(Categories, {through: 'ProductCategories'})
```

Reply:
That's my B - added changes